packages/foam-vscode/static/dataviz/graph.js

```diff
@@ -223,12 +253,34 @@ function initDataviz(channel) {
     .onBackgroundClick(event => {
       Actions.selectNode(null, event.getModifierState('Shift'));
     });
+    const gui = new dat.gui.GUI();
+    gui.add(model, 'View', {Default: "default", Tags: "tags"})
```

Comment:
same as above, let's use the JS naming convention

Reply:
Fixed.